lms/djangoapps/course_api/blocks/transformers/student_view.py

```diff
@@ -59,14 +59,14 @@ def collect(cls, block_structure):
                 cls.STUDENT_VIEW_MULTI_DEVICE,
                 supports_multi_device,
             )
-            if getattr(block, 'student_view_data', None):
-                student_view_data = block.student_view_data()
-                block_structure.set_transformer_block_field(
-                    block_key,
-                    cls,
-                    cls.STUDENT_VIEW_DATA,
-                    student_view_data,
-                )
+            # if getattr(block, 'student_view_data', None):
```

Comment:
Is there any specific reason to keep this comment. Also this comment is changed as well

```block.student_view_data() ``` .==> ```block.student_view_data({}) ```